lib/elixir/lib/enum.ex

```diff
@@ -1653,6 +1653,9 @@ defmodule Enum do
   It assumes that the sample being returned can fit into memory;
   the input `enumerable` doesn't have to, as it is traversed just once.
 
+  If a range is passed into the function it will not be expanded so it
```

Comment:
I'm not sure that we should mention this, it's an optimization that may or may not be here.
And it's better to use `Kernel.rand/1` from now on.